### ISSUE TITLE
Don't stop gracefully

### ISSUE THREAD

**Issue by ylecuyer** [djfarrelly/MailDev#118]
When using with docker and docker-compose, the container for maildev takes more time to stop and terminates with status code 137:
```
^CGracefully stopping... (press Ctrl+C again to force)
Stopping attendize_web_1 ... done
Stopping attendize_php_1 ... done
Stopping attendize_mail_1 ... done
Stopping attendize_db_1 ... done
php_1  | [11-May-2016 15:27:18] NOTICE: Terminating ...
php_1  | [11-May-2016 15:27:18] NOTICE: exiting, bye-bye!
db_1   | LOG:  received smart shutdown request
db_1   | LOG:  autovacuum launcher shutting down
db_1   | LOG:  shutting down
db_1   | LOG:  database system is shut down
attendize_web_1 exited with code 0
attendize_php_1 exited with code 0
attendize_db_1 exited with code 0
attendize_mail_1 exited with code 137
```

**Comment by djfarrelly** [comment]
Hi @ylecuyer! I'd love to improve this, but I'm not sure how to go about fixing this, do you have any suggestions?

**Comment by ylecuyer** [comment]
@djfarrelly maybe you should add a SIGTERM handler.

see https://docs.docker.com/engine/reference/commandline/stop/
and http://joseoncode.com/2014/07/21/graceful-shutdown-in-node-dot-js/

**Comment by djfarrelly** [comment]
This one is addressed in 0.13.3 - thanks for the nudge @ylecuyer!
